### ISSUE TITLE
use_bucket_encryption is not used

### ISSUE THREAD

**Issue by brnl** [youngfeldt/terraform-aws-backend-s3#3]
Hi @youngfeldt,

I tried to understand your module to see if I could use it.

Am I reading over it or is the variable 'use_bucket_encryption' not used in any way? And thus encryption is not enabled?

Thanks.

**Comment by youngfeldt** [comment]
Correct.   If you see the bottom of the Readme it was/is still a ToDo.   
I should share that since writing this, i've moved away from it in favor of a make file that managed the backend for me.    I'll make time later this week to share in gist.github and will note here.   I think I will deprecate this as it has been more of a hassle than a help.

**Comment by youngfeldt** [comment]
well, as others have found it useful, i'll finish the encryption part.

**Comment by brnl** [comment]
Sorry I missed that ToDo section.

I'm not sure if this module is what I need since the lifecycle { prevent_destroy = true } is giving me a hard time testing. It prevents me from destroying anything. Did you build a separate 'terraform project' for this and then just pointed your other backends to this bucket and dynamodb? Or how do you implement this in your projects?

>  I'll make time later this week to share in gist.github and will note here. I think I will deprecate this as it has been more of a hassle than a help.

I would love to see how you did that!
